libdenaro/src/helpers/currencyhelpers.cpp

```diff
@@ -1,5 +1,6 @@
 #include "helpers/currencyhelpers.h"
 #include <iomanip>
+#include <iostream>
```

Comment:
replace this with `#include <libnick/app/aura.h>`

but place it above `#include <libnick/helpers/stringhelpers.h>`